lite/kernels/arm/CMakeLists.txt

```diff
@@ -64,6 +64,7 @@ add_kernel(reduce_max_compute_arm ARM extra SRCS reduce_max_compute.cc DEPS ${li
 add_kernel(sequence_expand_compute_arm ARM extra SRCS sequence_expand_compute.cc DEPS ${lite_kernel_deps} math_arm)
 add_kernel(im2sequence_compute_arm ARM extra SRCS im2sequence_compute.cc DEPS ${lite_kernel_deps} math_arm)
 add_kernel(sequence_pool_compute_arm ARM extra SRCS sequence_pool_compute.cc DEPS ${lite_kernel_deps} math_arm)
+add_kernel(sequence_pool_grad_compute_arm ARM extra SRCS sequence_pool_grad_compute.cc DEPS ${lite_kernel_deps} math_arm)
```

Comment:
should add `train` kernel, instead of `extra` kernel as https://github.com/PaddlePaddle/Paddle-Lite/blob/7af1a25865d03fe762d8f2478e84539d439fbf7b/lite/kernels/arm/CMakeLists.txt#L103